packages/lib/gpc/test/gpcCompile.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { CircuitSignal } from "@pcd/gpcircuits";
```

Comment:
Not sure why this is involved in this PR.

Reply:
removed unrelated changes